radare2.spec

```diff
@@ -1,15 +1,30 @@
+%global         gituser         radare
+%global         gitname         radare2
+#%global        commit          a093958b6d24015d82782eb20a2e10d8f4afcd85
```

Comment:
Bug, RPM `#%` still evaluates values commented out. Go for `#%%`, the same applies later.
Why `%global`?